src/js/edu-benefits/actions/index.js

```diff
@@ -124,9 +129,20 @@ export function submitForm(data) {
     })
     .then(res => {
       if (res.ok) {
+        
+        window.dataLayer.push({
+          event: 'edu-submission-successful',
+          //submission-id: If you want to log anything here
+        });
+        
         return res.json();
       }
 
+      window.dataLayer.push({
+        event: 'edu-submission-failed',
+        //submission-error: If you want to log anything here
```

Comment:
It could be helpful to log the response in case of an error, but I'm pretty sure we're already logging that elsewhere, so probably not necessary to include in GA. @markolson can verify, but I would say we don't need to include it here for now.